mediaflow_proxy/extractors/fastream.py

```
@@ -0,0 +1,32 @@
+
+
+
+class FastreamExtractor(BaseExtractor):
+    """Fastream URL extractor."""
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.mediaflow_endpoint = "hls_manifest_proxy"
+
+    async def extract(self, url: str, **kwargs) -> Dict[str, Any]:
+        #Init headers needed for the request.
+        headers  = {'Accept': '*/*', 'Connection': 'keep-alive','Accept-Language': 'en-US,en;q=0.5','Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8', 'user-agent': 'Mozilla/5.0 (X11; Linux x86_64; rv:138.0) Gecko/20100101 Firefox/138.0'}
```

Ruff (0.11.9)

Dictionary key literal `'Accept'` repeated

(F601)